modules/java/HTTPDestination.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Created on 6/29/15.
+ */
```

Comment:
Please remove this comment above.
Git history will show when this file was added first.